src/BlockchainReader/Bitcoin.test.ts

```diff
@@ -38,50 +38,52 @@ describe('Bitcoin.blockToPoetAnchors', async should => {
 
     const anchorsIpfsHashes = poetAnchors.map(poetAnchor => poetAnchor.ipfsDirectoryHash)
 
+    const given = 'a test block with Po.et anchors, blockToPoetAnchors'
```

Comment:
blockToPoetAnchors is already covered in the describe() so it seems redundant here.

Reply:
fixed!